server/src/Transactions/transaction.controller.ts

```diff
@@ -10,25 +10,25 @@ import {
   Delete,
   Patch,
   UseGuards,
-} from '@nestjs/common';
-import { AuthGuard } from '@nestjs/passport';
-import { Response } from 'express';
-import { User } from 'src/Auth/Auth.interfaces';
-import { GetUser } from 'src/Auth/get-user.decorator';
-import { ValidateObjectId } from 'src/shared/pipes/validate-object-id.pipes';
-import { MongoDBID } from 'src/shared/types';
-import { TransactionDTO } from './dto/transaction.dto';
-import { IncomeOrExpense } from './transaction.interfaces';
-import { TransactionServices } from './transaction.service';
+} from "@nestjs/common";
```

Comment:
Please ignore all the double quote changes..

The only change I made on this file is from user._id to just 'user' in the get all expenses controller.